src/redis-cli.c

```diff
@@ -3467,7 +3476,7 @@ static void repl(void) {
                 }
 
                 elapsed = mstime()-start_time;
-                if (elapsed >= 500 &&
+                if (elapsed >= config.print_duration_threshold &&
```

Comment:
A few thoughts:

* If we print the duration in seconds (like `(0.02s)`) the option should also take the value in the same unit. (We can parse it and store it as milliseconds internally.)
* The option name is a bit complicated. (I don't have a better idea. The only idea  I can come up with now is variants like `--print-elapsed-if-gt` which is not really better.)
* Do you really need to tweak the value that much? Perhaps you just want to print the elapsed time for all commands? Maybe we can the existing `--verbose` flag, which doesn't do almost anything at all. (All it does it affects some printout in `--cluster rebalance`.)

```suggestion
                if ((elapsed >= 500 || config.verbose) &&
```

Reply:
Yean, the name is too long to use.

You are right, actually i want to print the elapsed time for all commands. verbose thing seem fine, not sure which is more versatile though, I actually like the small diff too